x-pack/plugins/observability/public/pages/alert_details/alert_details.tsx

```diff
@@ -36,6 +37,10 @@ const defaultBreadcrumb = i18n.translate('xpack.observability.breadcrumbs.alertD
   defaultMessage: 'Alert details',
 });
 
+export const LOG_DOCUMENT_COUNT_RULE_TYPE_ID = 'logs.alert.document.count';
```

Comment:
It is possible to import them from the `infra` plugin, but I don't know if this is advisable. We are importing types from observability plugin into `infra` plugin. This would mean importing both ways. Ideally we should define these in `observability` and import them in `infra` plugin, but I am not entirely sure. Do you think this could be an issue or not really?

Reply:
I will leave it as is right now. We can improve it later.